index.js

```diff
@@ -122,11 +134,14 @@ var publicAgent = {
     return agent.addTransactionLabel(key, value);
   },
 
-  start: function(projectConfig) {
+  startAgent: function(projectConfig) {
     if (this.isActive()) { // already started.
-      agent.logger.warn('Calling start on already started agent.' +
-        'New configuration will be ignored.');
-      return this;
+      var message = 'Cannot call start on an already started agent.';
```

Comment:
With the change I made to align the trace agent with the debug agent with respect to forcing a new agent to be started, this comment is only used once.